healthcheck/settings/dev.py

```diff
@@ -1,8 +1,11 @@
 from healthcheck.settings.base import *  # noqa: F403
+import dj_database_url
 
-SECRET_KEY = env.str("SECRET_KEY")  # noqa: F405
+SECRET_KEY = env.str("SECRET_KEY", "testsecretkey")  # noqa: F405
 DEBUG = env.bool("DEBUG", True)  # noqa: F405
 DATABASES = {
-    "default": env.db(),  # noqa: F405
+    "default": dj_database_url.config(
+        "postgres://postgres:postgres@localhost:5432/healthcheck"
+    )
```

Comment:
We already have `django-environ` for settings, so we can use that instead of adding the additional dependancy